.github/pull_request_template.md

```diff
@@ -1,9 +1,13 @@
 - Link to issue this resolves
+#<issuelink>
 
 - What I did
 
+
 - How I did it
 
+
 - How to verify it
 
+
 - One line description for the changelog
```

Comment:
I don't think we need all of these. Make it simpler.

`Fixes #..`

Example: https://github.com/google/new-project/blob/master/.github/PULL_REQUEST_TEMPLATE.md